pkg/cache/v2/cache.go

```diff
@@ -94,7 +126,29 @@ type RawResponse struct {
 	marshaledResponse atomic.Value
 }
 
+// RawDeltaResponse is a pre-serialized xDS response that utilizes the delta discovery request/response objects.
+type RawDeltaResponse struct {
+	// Request is the latest delta request on the stream.
+	DeltaRequest *discovery.DeltaDiscoveryRequest
+
+	// SystemVersionInfo is present for holding the current applied response version but generally used debugging purposes only.
```

Comment:
This sentence reads a bit weird? `SystemVersion holds the currently applied response system version and should be used for debugging purposes only`